File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.cs
using System;
using System.Collections.Generic;

public class Solution {
    public List<int> getMinMax(int[] arr) {
        // code here
        List<int> res = new List<int>();
        int max = arr[0];
        int min=arr[0];
        for(int i =1; i< arr.Length;i++){
            max = Math.Max(max, arr[i]);
            min= Math.Min(min, arr[i]);
        }
        
        res.Add(min);
        res.Add(max);
        return res;
    }
}

File: Difficulty: Basic/Min and Max in Array/min-and-max-in-array.java
class Solution {
    public ArrayList<Integer> getMinMax(int[] arr) {
        // code Here
        ArrayList<Integer> res= new ArrayList<>();
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        
        for(int i=0; i<arr.length;i++){
            
            if(arr[i]<min){
                min = arr[i];
            }
            if(arr[i]> max){
                max = arr[i];
            }
            
        }
        
        res.add(min);
        res.add(max);
        return res;
        
        
    }
}
